ibis/expr/operations.py

```diff
@@ -80,40 +322,25 @@ def blocks(self):
 
 
 class UnboundTable(PhysicalTable):
-
-    def __init__(self, schema, name=None):
-        if name is None:
-            name = genname()
-        TableNode.__init__(self, [schema, name])
-        HasSchema.__init__(self, schema, name=name)
+    schema = rlz.schema
+    name = rlz.optional(rlz.instanceof(six.string_types), default=genname)
```

Comment:
Can you document that `callable`s are allowed for defaults if you haven't already?

Reply:
Done.